src/linter/LinterContext.ts

```diff
@@ -58,13 +58,18 @@ export interface TranspileResult {
 
 export interface LinterOptions {
 	rootDir: string;
-	namespace?: string;
 	filePatterns?: FilePattern[];
 	ignorePattern?: FilePattern[];
 	reportCoverage?: boolean;
 	includeMessageDetails?: boolean;
 	configPath?: string;
 	ui5ConfigPath?: string;
+	namespace?: string;
+
+	relFsBasePath?: string;
```

Comment:
Is this really the right interface for those options? It is also used by `lintProject` and `lintFile`, but the options are not considered there, which is confusing.

Reply:
Ah, nice catch! Thanks!